### PR TITLE
CTCTRADERS-1028 | Updating seals pages to use CheckMode when loaded f…

### PR DESCRIPTION
**Pull request by PSouthern**
…rom CYA and seals-information pages

### CODE REVIEW

File: app/utils/AddSealHelper.scala [hunk 1]
```diff
@@ -35,7 +35,7 @@ class AddSealHelper(userAnswers: UserAnswers) {
           actions = List(
             Action(
               content            = msg"site.edit",
-              href               = SealIdDetailsController.onPageLoad(lrn, sealIndex, mode).url,
+              href               = SealIdDetailsController.onPageLoad(lrn, sealIndex, CheckMode).url,
```
Comment (chris-ianson) [inline app/utils/AddSealHelper.scala:38]: Can the mode argument be removed from this method (don't think it's used anymore, it's always in CheckMode)

File: app/utils/AddSealHelper.scala [hunk 2]
```diff
@@ -62,7 +62,7 @@ class AddSealHelper(userAnswers: UserAnswers) {
         actions = List(
           Action(
             content            = msg"site.edit",
-            href               = SealsInformationController.onPageLoad(lrn, mode).url,
+            href               = SealsInformationController.onPageLoad(lrn, CheckMode).url,
```
Comment (chris-ianson) [inline app/utils/AddSealHelper.scala:65]: Can the mode argument be removed from this method (don't think it's used anymore, it's always in CheckMode)

**Review by chris-ianson — CHANGES_REQUESTED** [review]
Can the mode argument be removed from the methods (don't think it's used anymore, it's always in CheckMode)

**Comment by PSouthern** [comment]
> Can the mode argument be removed from the methods (don't think it's used anymore, it's always in CheckMode)

Resolved now @chris-ianson

**Review by alexflav23 — APPROVED** [review]

**Review by chris-ianson — APPROVED** [review]

**Pull request merged**